news/_posts/translation/2020-12-09-do-you-speak-prestashop-november-2020.md

```diff
@@ -0,0 +1,121 @@
+
+In just a month, the Hebrew project went from 63% to 83% translated and validated, an increase of 20%! What a great evolution! :scream: Congratulations! Special thanks to [danielshapiro9](https://crowdin.com/profile/danielshapiro9) who has joined us just about a month ago and who has already translated 26,091 words. Thank you for your involvement! And of course, thank you to the other translators of the Hebrew project!
+
+### :two_hearts: Special thanks to nanosim2
+ 
+Many people subscribe to the project(s) of their choice each month, but not so many start translating straight away. For that reason, we would like to thank the ones who rushed into the effort immediately! So lots of love to the dedicated [nanosim2](https://crowdin.com/profile/nanosim2)!
+
+
+## A few stats
+ 
+* **39 members** were active on the project this month.
+* A total of **37,492 words have been translated** and **37,821 validated**.
+* All this in **24 different languages**.
+ 
+Thank you for your involvement!
+
+## Top contributors
+ 
+A lot of you are working every day on Crowdin to have PrestaShop available in many languages, and PrestaShop can't thank you enough for your dedication! Here are the 10 most active translators and proofreaders for November 2020.
+ 
+#### Top 10 translators in November:
+ 
+| |Translator | Language | # Words
+|-|---------- | -------- | ----------------
+| 1. | דניאל שפירא (danielshapiro9) | Hebrew | 26,091
+| 2. | Zoran Tejic (zotamal) | Serbian (Latin) | 1,690
+| 3. | Odelia Yechiel (OdeChan) | Hebrew | 1,495
+| 4. | Benjamin Gantikow (bbbenjie) | German | 1,276
+| 5. | Ronny (rbuelund) | Danish | 1,051
+| 6. | Gabriel Tenita (ggedamed) | Romanian | 962
+| 7. | SeongHyeon Cho (jaymz9634) | Korean | 664
+| 8. | mirmal | Esperanto | 613
+| 9. | Sretko Devič (Chico) | Slovenian | 429
+| 10. | jbonello | Maltese | 384
+ 
+
+#### Top 10 proofreaders in November:
+ 
+| | Proofreader | Language | # Words
+|-| ---------- | -------- | ----------------
+| 1. | דניאל שפירא (danielshapiro9) | Hebrew | 29,380
+| 2. | Zoran Tejic (zotamal) | Serbian (Latin) | 1,704
+| 3. | Ronny (rbuelund) | Danish | 1,051
+| 4. | Gabriel Tenita (ggedamed) | Romanian | 962
+| 5. | Benjamin Gantikow (bbbenjie) | German | 778
+| 6. | SeongHyeon Cho (jaymz9634) | Korean | 602
+| 7. | Sretko Devič (Chico) | Slovenian | 429
+| 8. | Adeko Webdesign & Development (Adeko) | Dutch | 420
+| 9. | Cha (cafetango) | Chinese Traditional | 393
+| 10. | Marcin Orzechowski (Martinovy) | Polish | 210
+ 
+Congrats, and welcome to our new top contributors! :clap:
+ 
+Remember, you can see who's been contributing to our translation project thanks to the [Translators page](http://translators.prestashop.com/).
+
+ 
+## Complete translations
+ 
+#### Fully translated languages
+ 
+At the end of November, PrestaShop 1.7.7 was fully available (= 100% translated and validated) in **5 languages**:
+ 
+| **Chinese traditional** | **Danish** | English | Portuguese, Brazil |
```

Comment:
Did you want to make a table? Then it must be on a single line.

Reply:
Yes, thank you!